scripts/mods/msu/persistent_data_manager.nut

```diff
@@ -0,0 +1,97 @@
+this.persistent_data_manager <- {
+	m = {
+		ModConfigPath = "mod_config/",
```

Comment:
Perhaps rename folder to msu_mod_config so that people know the folder is created and handled by msu.

Reply:
I disagree, the folder is made by bbparser and has subfolders for each mod (such as mod_config/MSU/Keybinds/...). It'd be confusing.

Reply:
Fair enough.